homework.md

```diff
@@ -4,77 +4,77 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Mark-up Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Styling a web page
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+.h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A web programming language that can calculate and validate data
```

Comment:
### Side Note here that you have to take into consideration:

- Nowadays, JavaScript is an all-purpose programming language—meaning it runs across the entire software stack. The most popular application of JavaScript is on the client side (aka frontend), but since Node.js came on the scene, many people run JavaScript on the server side (aka backend) as well not forgetting that when React Native came along we can now write Mobile Applications using JavaScript finally with Electron also we can build Desktop Applications using JavaScript.

### Key to this question is not answer "what JavaScript does" cause that is a question on it's own but is to answer "what is JavaScript", e.g Dynamic programming Lang etc...

Reply:
Thanks, bro for shading more light to me.